README.md

```diff
@@ -198,5 +200,5 @@ Examples
    - https://github.com/ome/scripts
 
 You can find more examples by searching for
-"[openmicroscopy/omero-test-infra filename:.travis.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3A.travis.yml&type=Code)"
+"[ome/omero-test-infra filename:omero_plugin.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3Aomero_plugin.yml&type=Code)"
```

Comment:
This query returns nothing, did you mean https://github.com/search?q=omero-test-infra+filename%3Aomero_plugin.yml&type=Code ?